libats-messaging-datatype/src/main/scala/com/advancedtelematic/libats/messaging_datatype/Messages.scala

```diff
@@ -168,6 +168,9 @@ object Messages {
 
   final case class CommitManifestUpdated(namespace: Namespace, commit: Commit, releaseBranch: String, metaUpdaterVersion: String)
 
+  final case class AktualizrConfigChanged(namespace: Namespace, uuid: DeviceId, pollingSec: Int,
```

Comment:
Lets include some kind of timestamp when it was received

Reply:
Ok. Same for CommitManifestUpdated?

Reply:
@jochenschneider yes that would be great